service-commands/scripts/provision-dev-env.sh

```diff
@@ -113,12 +119,12 @@ function setup_profile() {
     echo 'export AUDIUS_REMOTE_DEV_HOST=$(curl -sfL -H "Metadata-Flavor: Google" http://metadata/computeMetadata/v1/instance/network-interfaces/0/access-configs/0/external-ip)' >> $HOME/.profile
     echo 'export AAO_DIR=$HOME/anti-abuse-oracle' >> $HOME/.profile
     echo 'export TN_DIR=$HOME/trusted-notifier-service' >> $HOME/.profile
-    echo '
-if [ -d ~/.aliases ]; then
-  for f in ~/.aliases/*; do
-    source $f
-  done
-fi' >> $HOME/.profile
+        
+    if [ -d $PROTOCOL_DIR/service-commands/scripts/.env/.aliases ]; then
```

Comment:
This `.env` directory is meant to be used as a standalone project. README can be found here:

* https://github.com/AudiusProject/audius-protocol/tree/master/service-commands/scripts/.env

If you would like to get onboarded, do please let me know and we can screenshare together to work out any kinks I may have missed!

The added benefit is that when troubleshooting calls into questioned a customized setup, we can use the same tooling to quickly revert to a "vanilla" install.